packages/calcite-components/src/components/loader/loader.tsx

```diff
@@ -21,6 +22,9 @@ export class Loader {
   /** Accessible name for the component. */
   @Prop() label!: string;
 
+  /** When `true`, and when the type is `determinate`, displays the progress value as a percentage */
+  @Prop({ reflect: true }) percentage = false;
```

Comment:
Could this be confused with `value`? The description mentions "percent". Maybe `displayType` with two values, defaulting to the current one? It's not quite the same but `percent` is a value of a similar property in Meter: https://developers.arcgis.com/calcite-design-system/components/meter/#api-reference-properties-rangeLabelType

Reply:
I can see that. WDYT about:

* `valueType: "number" | "percent"` (similar to [`meter#valueLabelType`](https://developers.arcgis.com/calcite-design-system/components/meter/#api-reference-properties-valueLabelType))
* `valueStyle: "number" | "percent"` (similar to [`input-time-zone#offsetStyle`](https://developers.arcgis.com/calcite-design-system/components/input-time-zone/#api-reference-properties-offsetStyle))

?